packages/create-expo/package.json

```
@@ -21,6 +21,9 @@
     "build",
     "template"
   ],
+  "engines": {
+    "node": ">=18"
```

```suggestion
    "node": ">=18.13.0"
```

This also includes the [`File`](https://nodejs.org/api/buffer.html#class-file) primitive in Node, which might come in handy when doing more advanced streaming.